deploy.cmd

```diff
@@ -0,0 +1,29 @@
+
+@echo off
+
+"tools\nuget\nuget.exe" "install" "FAKE.Core" "-OutputDirectory" "tools" "-ExcludeVersion" "-version" "4.4.2"
+
+:Build
+cls
+
+SET TARGET="CreatePackages"
+SET GITOWNER="octokit"
+SET GITPASSWORD=""
```

Comment:
`GITHUBPASSWORD` is used while publishing the release. You could use https://github.com/settings/tokens